classes/core/PKPApplication.inc.php

```diff
@@ -69,6 +69,7 @@
 define('WORKFLOW_TYPE_AUTHOR', 'author');
 
 use Illuminate\Database\Capsule\Manager as Capsule;
+use Illuminate\Database\Query\Builder;
```

Comment:
I don't think the changes to this class are necessary.

Reply:
Yep, this is cruft that remained from removed code.